packages/rollup-tests/src/ignored-tests.js

```diff
@@ -19,7 +19,10 @@ const ignoreTests = [
 
   "rollup@hooks@assigns chunk IDs before creating outputBundle chunks", // The `renderChunk` is called at parallel, collect chunk info to array is unstable.  https://github.com/rolldown/rolldown/issues/2364
   "rollup@form@non-empty-block-statement: do not remove non an empty block statement@generates es", // https://github.com/rolldown/rolldown/pull/3541#issuecomment-2649731213
-  "rollup@function@es5-class-called-without-new" // rolldown align directive rendering with esbuild
+  "rollup@function@es5-class-called-without-new", // rolldown align directive rendering with esbuild
+
+  // hyf0: We can align the deconflict logic with rollup, but it requires unnecessary logic and doesn't have payoff.
+  "rollup@form@deconflict-module-priority: prioritizes entry modules over dependencies when deconflicting",
```

Comment:
Since static import stmts are included now. `import { foo } from './foo.js'` in `entry.js` will cause symbol `foo` in `foo.js` participate in the deconflict early than others.

This test wants to ensure all entry's declared symbols get processed first. I feel unnecessary to put effort on this one.